src/lib/combat_achievements/master.ts

```diff
@@ -143,45 +143,45 @@ export const masterCombatAchievements: CombatAchievement[] = [
 	{
 		id: 2011,
 		name: 'Perfect Olm (Solo)',
-		desc: 'Kill the Great Olm in a solo raid without taking damage from any of the following: Teleport portals, Fire Walls, Healing pools, Crystal Bombs, Crystal Burst or Prayer Orbs. You also cannot let his claws regenerate or take damage from the same acid pool back to back.',
+		desc: 'Kill the Great Olm in a solo raid without taking damage from any of the following: Teleport portals, Fire Walls, Healing pools, Crystal Bombs, Crystal Burst or Prayer Orbs. You also cannot let his claws regenerate or take damage from the same acid pool back to back. (Party size required)',
 		type: 'perfection',
 		monster: 'Chambers of Xeric',
 		rng: {
 			chancePerKill: 44,
-			hasChance: 'Raids'
+			hasChance: data => data.type === 'Raids' && (data as RaidsOptions).users.length === 1
 		}
 	},
 	{
 		id: 2012,
 		name: 'Chambers of Xeric (Solo) Speed-Chaser',
-		desc: 'Complete a Chambers of Xeric (Solo) in less than 21 minutes.',
+		desc: 'Complete a Chambers of Xeric (Solo) in less than 21 minutes. (Party size required)',
 		type: 'speed',
 		monster: 'Chambers of Xeric',
 		rng: {
 			chancePerKill: 25,
-			hasChance: 'Raids'
+			hasChance: data => data.type === 'Raids' && (data as RaidsOptions).users.length === 1
 		}
 	},
 	{
 		id: 2013,
 		name: 'Chambers of Xeric (5-Scale) Speed-Chaser',
-		desc: 'Complete a Chambers of Xeric (5-scale) in less than 15 minutes.',
+		desc: 'Complete a Chambers of Xeric (5-scale) in less than 15 minutes. (Party size required)',
 		type: 'speed',
 		monster: 'Chambers of Xeric',
 		rng: {
 			chancePerKill: 33,
-			hasChance: 'Raids'
+			hasChance: data => data.type === 'Raids' && (data as RaidsOptions).users.length === 1
 		}
```

Comment:
Enforcing a party size of 1 (solo) for a 5 man achievement ?

As you haven't enforced this on any other of the size related ones for cox assume its a mistake